_episodes/09-conflict.md

```diff
@@ -323,8 +323,11 @@ Conflicts can also be minimized with project management strategies:
 - Discuss what order tasks should be carried out in with your collaborators so
   that tasks expected to change the same lines won't be worked on simultaneously
 - If the conflicts are stylistic churn (e.g. tabs vs. spaces), establish a
-  project convention that is governing and use code style tools (e.g.
-  `htmltidy`, `perltidy`, `rubocop`, etc.) to enforce, if necessary
+  project convention that is governing, use code style tools (e.g.
+  `styleR` or `Code > Reformat` in RStudio,
+  or [`pep8` in Python](https://swcarpentry.github.io/python-novice-gapminder/18-style/index.html#follow-standard-python-style-in-your-code),
```

Comment:
Since this particular section in the gapminder lesson mentions the google style guide for Python, it might be worth mentioning here too.

Reply:
This is a good point, because the Google style guide recommends to use the "[yapf](https://github.com/google/yapf/) auto-formatter". So, I suggest we either add a link to it to [python-novice-gapminder/18-style](https://github.com/swcarpentry/python-novice-gapminder/blob/gh-pages/_episodes/18-style.md#follow-standard-python-style-in-your-code), or don't mention either one here.